src/main/kotlin/com/terraformation/backend/customer/AppNotificationService.kt

```diff
@@ -349,6 +388,26 @@ class AppNotificationService(
     }
   }
 
+  private fun insertAcceleratorNotification(
+      localUrl: URI,
+      notificationType: NotificationType,
+      organizationId: OrganizationId,
+      renderMessage: () -> NotificationMessage,
+  ) {
+    val recipients = HashSet(userStore.fetchWithGlobalRoles())
```

Comment:
Should this be scoped to TF experts only?

Reply:
By TF Experts they mean all the current global roles, had clarified with Leann.
Not specifically the TF Expert role.

Reply:
Ah, I see, thanks.